tajo-rpc/src/test/java/org/apache/tajo/rpc/TestAsyncRpc.java

```diff
@@ -125,8 +125,11 @@ public void setUpRpcServer() throws Exception {
   public void setUpRpcClient() throws Exception {
     retries = 1;
 
-    client = new AsyncRpcClient(DummyProtocol.class,
-        RpcUtils.getConnectAddress(server.getListenAddress()), retries);
+    RpcConnectionPool.RpcConnectionKey rpcConnectionKey =
+          new RpcConnectionPool.RpcConnectionKey(
+              RpcUtils.getConnectAddress(server.getListenAddress()),
+              DummyProtocol.class, true);
+    client = new AsyncRpcClient(rpcConnectionKey, retries);
```

Comment:
It needs to acquire connection on this code block, or it can be initialized later on each test functions.